doc/user_guide/user_guide.md

```diff
@@ -174,3 +225,61 @@ While the `cp` command can rename the copied file in the target location you can
 bfsc cp foo.jar bfs:/drivers/
 bfsc cp bfs:/drivers/foo.jar .
 ```
+
+Command `cp` supports recursive copying:
+
+```shell
+bfsc cp -r Dir bfs:/
+```
+
+However there are some special cases explained in the following sections.
+
+##### Ambigue Entries in BucketFS
+
+BucketFS differes from the local file system as it may contain *ambigue* entries. An ambigue entry represents a regular file and a directory at the same time. Obviously you cannot download both flavors of such an entry to the local file system and BFSC will abort with an error message.
```

Comment:
```suggestion
BucketFS differs from the local file system as it may contain *ambigue* entries. An ambigue entry represents a regular file and a directory at the same time. Obviously you cannot download both flavors of such an entry to the local file system and BFSC will abort with an error message.
```